NPE when starting (Test) app

The following NPE occured on a local test.

## Bug description 
~~~
E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
    Process: cgeo.geocaching, PID: 2335
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:354)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: java.lang.NullPointerException: Attempt to get length of null array
        at cgeo.geocaching.storage.LocalStorage$MigrateTask.doInBackgroundInternal(LocalStorage.java:443)
        at cgeo.geocaching.storage.LocalStorage$MigrateTask.doInBackgroundInternal(LocalStorage.java:401)
        at cgeo.geocaching.utils.AbstractAsyncTaskWithProgress.doInBackground(AbstractAsyncTaskWithProgress.java:134)
        at android.os.AsyncTask$2.call(AsyncTask.java:333)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:764) 
~~~

## Reproduce 

Not sure what was the trigger.

--- a/main/src/cgeo/geocaching/storage/LocalStorage.java
+++ b/main/src/cgeo/geocaching/storage/LocalStorage.java
@@ -440,10 +440,13 @@ public final class LocalStorage {
 
                 //migrate existing Offline Log Image paths
                 if (legacyOfflineLogImagesDir.isDirectory()) {
-                    for (File offlineImage : legacyOfflineLogImagesDir.listFiles()) {
-                        if (offlineImage.isFile()) {
-                            displayProgress(offlineImage.getName());
-                            FileUtils.copy(offlineImage, ImageUtils.getFileForOfflineLogImage(offlineImage.getName()));
+                    final File[] files = legacyOfflineLogImagesDir.listFiles();
+                    if (files != null) {
+                        for (File offlineImage : files) {
+                            if (offlineImage.isFile()) {
+                                displayProgress(offlineImage.getName());
+                                FileUtils.copy(offlineImage, ImageUtils.getFileForOfflineLogImage(offlineImage.getName()));
+                            }
                         }
                     }
                 }

